src/NuProj.Package.Dev14/NuProj.Package.Dev14.csproj

```diff
@@ -17,7 +17,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NuProj.ProjectSystem</RootNamespace>
-    <AssemblyName>NuProj.ProjectSystem</AssemblyName>
+    <AssemblyName>NuProj.ProjectSystem14</AssemblyName>
```

Comment:
the dot before the number is fine. There's not much need to put the .0 at the end. VS does it because we actually ship new functionality in Updates so we need to differentiate. But we don't make breaking changes in VS in Updates so NuProj.ProjectSystem shouldn't ever need a 14.1 assembly that is side-by-side with a 14.0. Probably anyway.